docs/beyond_basics/101-168-dvc.rst

```diff
@@ -0,0 +1,1032 @@
+- the data is analyzed using various ML models in DVC pipelines
+
+This handbook section is structured in the form of a comparison.
+We demonstrate each step with DVC according to their tutorial, and then recreate a corresponding DataLad workflow for a direct comparison.
+If you want to, you can code along, or only browse through the direct comparisons of DVC and DataLad commands.
+Some familiarity with DataLad can be helpful, but if you have never used DataLad, footnotes in each section can point you relevant chapters for more insights on a command or concept.
+If you have never used DVC, `their technical docs <https://dvc.org/doc/command-reference>`_ or `collection of third-party tutorials <https://github.com/iterative/dvc.org/issues/1749>`_ can be of help for further questions.
+
+.. admonition:: If you are not a Git user
+
+   DVC relies heavily on Git workflows.
+   Understanding the DVC workflows requires a solid understanding of :term:`branch`\es, Git's concepts of `Working tree, Index ("Staging Area"), and Repository <https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository>`_, and some basic Git commands such as ``add``, ``commit``, and ``checkout``.
+   `The Turing Way <https://the-turing-way.netlify.app/welcome.html>`_ has an excellent `chapter on version control with Git <https://the-turing-way.netlify.app/reproducible-research/vcs.html>`_ if you want to catch up on some basics first.
+
+.. gitusernote::
+
+   Be mindful: DVC comes with a range of commands and concepts that have the same names, but differ in functionality to their Git namesake.
+   Make sure to read the `DVC documentation <https://dvc.org/doc/command-reference>`_ for each command to get more information on what it does.
+
+Setup
+^^^^^
+
+The `DVC tutorial <https://realpython.com/python-data-version-control/>`_ comes with a pre-made repository that is structured for DVC machine learning analyses. If you want to code along, `the repository <https://github.com/datalad-handbook/data-version-control.git>`_ needs to be :term:`fork`\ed (requires a GitHub account) and cloned from your own fork [#f1]_.
+
+.. runrecord:: _examples/DL-101-168-101
+   :workdir: DVCvsDL
+   :language: console
+
+   ### DVC
+   # please clone this repository from your own fork when coding along
+   $ git clone git@github.com:datalad-handbook/data-version-control.git DVC
+
+The resulting Git repository is already pre-structured in a way that aids DVC ML analyses: It has the directories ``model`` and ``metrics``, and a set of Python scripts for a machine learning analysis in ``src/``.
+
+.. runrecord:: _examples/DL-101-168-102
+   :workdir: DVCvsDL
+   :language: console
+
+   ### DVC
+   $ tree DVC
+
+For a comparison, we will recreate a similarly structured DataLad dataset.
+For greater compliance with DataLad's :ref:`YODA principles <yoda>`, the dataset structure will differ marginally in that scripts will be kept in ``code/`` instead of ``src``.
```

Comment:
```suggestion
For greater compliance with DataLad's :ref:`YODA principles <yoda>`, the dataset structure will differ marginally in that scripts will be kept in ``code/`` instead of ``src/``.
```